sdk/core/loader/types.h

```diff
@@ -905,8 +899,10 @@ namespace loader
 
 	/**
 	 * The header of an export table.
+	 *
+	 * This is followed by an array of `ExportEntry`.
```

Comment:
I suppose I understand why C++ doesn't do FAMs, and yet...

Reply:
It does (at least, GNU C++), but unfortunately the ABI doesn't put these entries in without padding.